query-node/mappings/src/content/video.ts

```diff
@@ -184,9 +225,15 @@ export async function content_ContentUpdated(ctx: EventContext & StoreContext):
   })
 
   if (video) {
-    const contentMetadata = newMeta.isSome ? deserializeMetadata(ContentMetadata, newMeta.unwrap()) : undefined
-
-    await processUpdateVideoMessage(ctx, video, contentMetadata?.videoMetadata || undefined, contentUpdatedEventData)
+    const appAction = newMeta.isSome ? deserializeMetadata(AppAction, newMeta.unwrap()) : undefined
+    if (appAction && 'signature' in appAction) {
```

Comment:
```suggestion
    if (appAction) {
```
If the signature is not present is should be handled later, with a clear warning message. The fact that we could deserialize the message as `AppAction` already means it was supposed to be deserialized this way (due to a new required `appId = 999` prop)